packages/block-library/src/gallery/tiles.native.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Children } from '@wordpress/element';
+
+const containerStyle = {
+	flexDirection: 'row',
+	flexWrap: 'wrap',
+};
+
+const tileStyle = {
+	overflow: 'hidden',
+	flexDirection: 'row',
+	alignItems: 'center',
+	borderColor: 'transparent',
+};
+
+function Tiles( props ) {
+	const {
+		columns,
+		children,
+		groutSpacing = 10,
```

Comment:
This one looked like a typo until I decided to check the dictionary first (TIL 😁). I think `spacing` works as well and might be clearer/shorter.

Reply:
Perhaps I took the metaphor a bit too far :laughing:. I changed this to `spacing`.

Reply:
I agree that spacing work better. I should say though, when I found out what grout means I had a lol moment, I appreciate how creative this metaphor is :D